jetty-util/src/test/java/org/eclipse/jetty/util/thread/ReservedThreadExecutorTest.java

```diff
@@ -277,7 +305,7 @@ public void stressTest() throws Exception
         reserved.setIdleTimeout(0, null);
         reserved.start();
 
-        final int LOOPS = 1000000;
+        final int LOOPS = 2000000;
```

Comment:
This seems too large to me, I don't think 2 mil. executions will show anything more than 2k.

Reply:
reduced a bit.  I was really just wanting the test to run long enough that the JIT would have done some changes